splitter.config.js

```diff
@@ -17,7 +17,7 @@ module.exports = {
   entry: resolve('src/Nightwatch.fsproj'),
   outDir: resolve("out"),
   babel: {
-     //   presets: [["es2015", { modules: false }]],
+    plugins: ["@babel/plugin-transform-modules-commonjs"],
```

Comment:
In the example, I have you I use `@babel/plugin-transform-modules-commonjs` because it was needed for `node.js` support

I don't know, if this is needed for React Native env.

Reply:
ah ok. let me remove it for now